src/sdk/bucketspace_fs.js

```diff
@@ -98,16 +99,41 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             return account;
         } catch (err) {
             dbg.error('BucketSpaceFS.read_account_by_access_key: failed with error', err);
-            throw new RpcError('NO_SUCH_ACCOUNT', `Account with access_key not found`, err);
+            if (err.code === 'ENOENT') {
+                throw new RpcError('NO_SUCH_ACCOUNT', `Account with access_key not found.`, err);
+            }
+            throw new RpcError('NO_SUCH_ACCOUNT', err.message);
+        }
+    }
+
+    async validate_account_schema(account) {
+        if (!account.name || !account.email) {
+            throw new RpcError('MISSING_ACCOUNT_INFO', 'Missing account info.');
+        }
+        if (!account.access_keys[0] || !account.access_keys[0].access_key || !account.access_keys[0].secret_key) {
+            throw new RpcError('INVALID_ACCESS', 'Missing account info.');
+        }
+        if (!account.nsfs_account_config || account.nsfs_account_config.uid === undefined ||
+            account.nsfs_account_config.gid === undefined || !account.nsfs_account_config.new_buckets_path) {
+            throw new RpcError('MISSING_NSFS_ACCOUNT_CONFIGURATION', 'Missing account configuration.');
         }
     }
 
+    async validate_bucket_schema(bucket) {
+        if (!bucket.name || !bucket.system_owner || !bucket.bucket_owner) {
+            throw new RpcError('INVALID_BUCKET_INFO', 'Missing bucket info.');
+        }
+        if (!bucket.path || !bucket.creation_date) {
+            throw new RpcError('INVALID_BUCKET_STATE', 'Missing bucket storage details.');
```

Comment:
If I see an error "'Missing bucket storage details." how can I know how to fix it?